app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+   # @user = User.new(:username => params[:username], :email => params[:email], :password => params[:password])
```

Comment:
Please remove all the code commented. Use comments just if you want to describe something.

Reply:
commented codes were removed, helper for email included as f.email_field. However I do not understand the helper for name? Note that in our project we used username not name so f.label will be username and for the name it will still be f.text_field(:username) because it was included in @user..